README.md

```diff
@@ -1,6 +1,4 @@
-# Home Assistant SRG SSR Weather integration
-
-> IMPORTANT: The API has been deprecated in favour of a different one and I haven't updated yet. As such, this integration doesn't currently work!
+# Home Assistant SRG SSR SRF-Weather integration
```

Comment:
Now that the integration is using the SRF Meteo API I think we should rename it accordingly.
What do you think?

Reply:
Agreed! The question is how we should name it exactly, it seems that SRG SSR is not very consistent either. Depending on where I look at, it says "SRF Weather [REST API]", "SRF-Weather" or "SRF Meteo [API]". I guess "SRF Weather" is the new official name since that is what is used as title in the developers documentation.

How far would you go? Rename git repository?

Reply:
> I guess "SRF Weather" is the new official name since that is what is used as title in the developers documentation.

let's go with that then!



> How far would you go? Rename git repository?

yeah, I would like to rename the entire repository as well, just to keep things clear.